code/modules/newscaster/obj/newscaster.dm

```diff
@@ -54,17 +54,16 @@
 	name = "security newscaster"
 	is_security = TRUE
 
-/obj/machinery/newscaster/New()
+/obj/machinery/newscaster/Initialize(mapload)
+	armor = list(melee = 50, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0, fire = 50, acid = 30)
+	. = ..()
```

Comment:
is there a reason `armor` got brought up before the parent call here?

Reply:
There is!
`armor` being a list is a shorthand that only valid before `Initialize` runs and converts it to be a `/datum/armor` proper. Otherwise you get runtimes trying to get `.brute` field of a list.

Now, why is this set in `New` / `Initialize` and not on the datum itself? I have no idea, it was moved here in #14527 with no comment.
Moving it back to datum does not appear to break anything, so let me do just that.